doc/nrf/libraries/zigbee/shell.rst

```diff
@@ -384,6 +384,168 @@ Example:
 
 ----
 
+.. _zcl_cmd:
+
+zcl cmd
+=======
+
+Send a generic ZCL command to the remote node.
+
+.. parsed-literal::
+   :class: highlight
+
+   zcl cmd [-d] *h:dst_addr* *d:ep* *h:cluster* [-p *h:profile*] *h:cmd_ID* [-l *h:payload*]
+
+.. note::
+    By default, the profile is set to Home Automation Profile, and the payload is empty.
+
+    The payload requires the **little-endian** byte order.
+
+    To send a request using binding table entries, set ``dst_addr`` and ``ep`` to ``0``.
+
+Send a generic ZCL command with ID ``cmd_ID`` and payload ``payload`` to the cluster ``cluster``.
+The cluster belongs to the profile ``profile``, which resides on the endpoint ``ep`` of the remote node ``dst_addr``.
+Optional default response can be set with ``-d``.
```

Comment:
I'm not sure if this can be `set`. I'd propose to write `requested` or `enabled` instead.

Reply:
Changed to `requested`